package/MDAnalysis/core/universe.py

```diff
@@ -407,6 +407,17 @@ def __init__(self, topology=None, *coordinates, all_coordinates=False,
             self._trajectory.add_transformations(*transformations)
 
         if guess_bonds:
+            warnings.warn(
+                "Bond guessing through the `guess_bonds` keyword is deprecated"
+                " and will be removed in MDAnalysis 3.0. "
+                "Instead, pass 'bonds', 'angles', and 'dihedrals' to "
+                "the `to_guess` keyword in Universe for guessing these. "
```

Comment:
add a 
```reST
.. deprecated:: 2.8.0
   Will be removed in 3.0.0. Pass pass into Context creation or Universe.guess_TopologyAttrs.
```
under each of the deprecated keyword args `fudge_factor`, `vdwradii`, `lower_bound`. Link to `Context` docs and `guess_TopologyAttrs`.

Reply:
Be specific about where the kwargs should go, either Context or guess_TopologyAttrs.

Reply:
A few extra thoughts:  
1.	The order of the attributes in `['bonds', 'angles', 'dihedrals']` is important because later attributes depend on the earlier ones. We may want to implement a function that automatically reorders the guessing sequence to ensure proper dependencies are respected?
	2.	The correct usage should be `force_guess=['bonds', 'angles', 'dihedrals']` instead of `to_guess`. Refer to https://github.com/MDAnalysis/mdanalysis/issues/4759

Reply:
I think the connectivity topology attrs do have dependencies built in, happily -- there's an existing test that guesses angles without bonds (https://github.com/MDAnalysis/mdanalysis/blob/d2729f71809f1f5fc2d3644ef48ee351e3b28f3d/testsuite/MDAnalysisTests/guesser/test_default_guesser.py#L167-L173). I think that means we don't need to specify the dependencies in order. Hmm... I should add a test that guessing angles/dihedrals does respect the `vdwradii`, etc. keywords used to control bond guessing though.